packages/api/src/controllers/webhook.test.js

```diff
@@ -47,6 +47,7 @@ beforeAll(async () => {
     kind: "webhook",
     events: ["stream.started"],
     url: "https://winter-darkness-88ea.livepeer.workers.dev/",
+    sharedSecret: "keyboardcat",
```

Comment:
I'm wondering if you mean 
![music cat](https://media.giphy.com/media/Hcw7rjsIsHcmk/giphy.gif)
or
![coding cat](https://media.tenor.com/images/d98e76e3930cf171cc39e301c9e974af/tenor.gif)

Reply:
haha. second one, that first cat knows what it's doing. it has to be random :D